ietf-te-types.yang

```diff
@@ -2111,89 +2198,231 @@ module ietf-te-types {
        Protocol-Traffic Engineering (RSVP-TE)";
   }
 
-  // CHANGE NOTE: The description of the identity path-metric-type
-  // has been updated in this module revision
+  // CHANGE NOTE: The path-metric-optimization-type base identity
+  // has been added in this module revision
   // RFC Editor: remove the note above and this note
-  identity path-metric-type {
-    description
-      "Base identity for the path metric type.
-      
-      Derived identities SHOULD describe the unit and maximum value
-      of the path metric types they define.";
-  }
-
-  identity path-metric-te {
-    base path-metric-type;
-    description
-      "TE path metric.";
-    reference
-      "RFC 3785: Use of Interior Gateway Protocol (IGP) Metric as a
-       second MPLS Traffic Engineering (TE) Metric";
-  }
-
-  identity path-metric-igp {
-    base path-metric-type;
+  identity path-metric-optimization-type {
     description
-      "IGP path metric.";
-    reference
-      "RFC 3785: Use of Interior Gateway Protocol (IGP) Metric as a
-       second MPLS Traffic Engineering (TE) Metric";
+      "Base identity used to define the path metric optimization
+      types.";
   }
 
-  // CHANGE NOTE: The reference for the identity path-metric-hop
+  // CHANGE NOTE: The link-path-metric-type base identity
   // has been added in this module revision
   // RFC Editor: remove the note above and this note
-  identity path-metric-hop {
-    base path-metric-type;
+  identity link-path-metric-type {
     description
-      "Hop path metric.";
-    reference
-      "RFC5440: Path Computation Element (PCE) Communication
-      Protocol (PCEP)";
+      "Base identity used to define the link and the path metric
+      types.
+      
+      The unit of the path metric value is interpreted in the
+      context of the path metric type and the derived identities
+      SHOULD describe the unit of the path metric types they
+      define.";
   }
 
+    // CHANGE NOTE: The link-metric-type base identity
+    // and its derived identities
+    // have been added in this module revision
+    // RFC Editor: remove the note above and this note
+    identity link-metric-type {
+      base link-path-metric-type;
+      description
+        "Base identity for the link metric types.";
+    }
 
-  identity path-metric-delay-average {
-    base path-metric-type;
-    description
-      "Average unidirectional link delay.";
-    reference
-      "RFC 7471: OSPF Traffic Engineering (TE) Metric Extensions";
-  }
+      identity link-metric-te {
```

Comment:
are there extra tabs?

Reply:
It's a convention I have seen elsewhere to make the derived identities look like as being nested after the base identity

I found this convention quite convenient when looking for the next base identity